public-api/src/integration/java/org/pmiops/workbench/google/BuildIapRequest.java

```diff
@@ -0,0 +1,62 @@
+package org.pmiops.workbench.google;
+
+import com.google.api.client.http.HttpRequest;
+import com.google.api.client.http.HttpRequestInitializer;
+import com.google.api.client.http.HttpTransport;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.auth.http.HttpCredentialsAdapter;
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.auth.oauth2.IdTokenCredentials;
+import com.google.auth.oauth2.IdTokenProvider;
+import com.google.common.base.Preconditions;
+import java.io.IOException;
+import java.util.*;
+
+public class BuildIapRequest {
```

Comment:
nit: please link back to the documentation where this came from

Reply:
Done thanks.